portfolio/pom.xml

```diff
@@ -23,7 +23,14 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.8.6</version>
+  	</dependency>
```

Comment:
Nit: formatting here looks weird. <dependency> start and end tags should be on the same column.